LEGO1/lego/legoomni/include/legocontrolmanager.h

```diff
@@ -26,10 +26,13 @@ class LegoControlManagerEvent : public LegoEventNotificationParam {
 	inline void SetClickedAtom(const char* p_clickedAtom) { m_clickedAtom = p_clickedAtom; }
 	inline void SetUnknown0x28(MxS16 p_unk0x28) { m_unk0x28 = p_unk0x28; }
 
+	MxS32 m_clickedObjectId; // 0x20
```

Comment:
Please use the existing getters `GetClickedObjectId` and `GetUnknown0x28` instead of making members public, thanks!